play/src/main/scala/org/lyranthe/prometheus/client/play/filters/PrometheusFilter.scala

```diff
@@ -0,0 +1,54 @@
+package org.lyranthe.prometheus.client.play.filters
+
+import org.lyranthe.prometheus.client.{play => _, _}
+import org.lyranthe.prometheus.client.histogram.LabelledHistogram
+import play.api.mvc._
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success, Try}
+
+class PrometheusFilter()(implicit val registry: Registry, executionContext: ExecutionContext) extends Filter {
```

Comment:
This will force the client code to have the `Registry` and the `ExecutionContext` as Beans in the context.

Much nicer I'd say to leave this up the client code. Example usage could be:
Ordinary Play Filter configuration:
```
class Filters @Inject()() extends HttpFilters {

  val filters = Seq(Filters.prometheusFilter)

}

object Filters {

  implicit val registry = Metrics.registry

  private val prometheusFilter = new PrometheusFilter()

}
```
or, that is nice and clean as well
```
class Filters @Inject()(prometheusFilter: PrometheusFilter) extends HttpFilters {

  val filters = Seq(prometheusFilter)

}
```
+
```
  @Provides
  @Singleton
  def prometheusFilter(): PrometheusFilter = {
    import scala.concurrent.ExecutionContext.Implicits.global
    implicit val registry = Metrics.registry
    new PrometheusFilter()
  }
```